packages/odyssey-storybook/src/components/odyssey-labs/Drawer/Drawer.stories.tsx

```diff
@@ -0,0 +1,454 @@
+          maxWidth: "1000px",
+          padding: "24px",
+          gridColumnGap: "24px",
+          gridTemplateColumns: "minmax(0, 2fr) auto",
+          animation: isVisible
+            ? "animate-drawer-open 225ms cubic-bezier(0, 0, 0.2, 1)"
+            : "animate-drawer-close 225ms cubic-bezier(0, 0, 0.2, 1)",
+        }}
+      >
+        <Box>
+          <h2>CSS Grid Layout</h2>
+          {gridStubText}
+        </Box>
+        {isVisible ? (
+          <Box>
+            <Drawer
+              {...props}
+              primaryCallToActionComponent={
+                <Button label="Primary" onClick={onClose} variant="primary" />
+              }
+              secondaryCallToActionComponent={
+                <Button
+                  label="Secondary"
+                  onClick={onClose}
+                  variant="secondary"
+                />
+              }
+              tertiaryCallToActionComponent={
+                <Button label="Cancel" onClick={onClose} variant="floating" />
+              }
+              onClose={onClose}
+              isOpen={isVisible}
+            />
+          </Box>
+        ) : null}
+      </Box>
+    );
+
+    return (
+      <>
+        {props.variant === "persistent" ? (
+          <>
+            <Button
+              label={isVisible ? "Close drawer" : "Open drawer"}
+              onClick={() => {
+                isVisible ? onClose() : onOpen();
+              }}
+              variant="primary"
+            />
+            {gridLayout}
+          </>
+        ) : (
+          <>
+            <Button label="Open drawer" onClick={onOpen} variant="primary" />
+            <Drawer
+              {...props}
+              primaryCallToActionComponent={
+                <Button label="Primary" onClick={onClose} variant="primary" />
+              }
+              secondaryCallToActionComponent={
+                <Button
+                  label="Secondary"
+                  onClick={onClose}
+                  variant="secondary"
+                />
+              }
+              tertiaryCallToActionComponent={
+                <Button label="Cancel" onClick={onClose} variant="floating" />
+              }
+              onClose={onClose}
+              isOpen={isVisible}
+            />
+          </>
+        )}
+      </>
+    );
+  },
+};
+
+export const Overlay: StoryObj<DrawerProps> = {
+  ...DefaultTemplate,
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Overlay blocks all other page actions and sits on top page content.",
+      },
+    },
+  },
+  args: {
+    children: drawerLongText,
+    title: "Okta Privileged Access",
+  },
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+    await step("open Default Drawer", async () => {
+      const buttonElement = canvas.getByText("Open drawer");
+      userEvent.click(buttonElement);
+      await waitFor(() => {
+        axeRun("Default Drawer");
```

Comment:
This needs to be `await`ed too

Reply:
I think adding it here causes an issue, I get: "'await' expressions are only allowed within async functions and at the top levels of modules"

Reply:
Sorry, the fix is a bit more than I suggested:
```
await waitFor(async () => { // <-- async prefix on the callback function parameter
  await axeRun("Default Drawer");  // <--- await here
});
```